app/src/main/java/e/josephmolina/saywhat/Modules/SayWhatApp.java

```diff
@@ -0,0 +1,19 @@
+package e.josephmolina.saywhat.Modules;
+
+import android.app.Application;
+
+import com.squareup.leakcanary.LeakCanary;
+
+public class SayWhatApp extends Application {
+    @Override
+    public void onCreate() {
+        super.onCreate();
+        if (LeakCanary.isInAnalyzerProcess(this)) {
+            // This process is dedicated to LeakCanary for heap analysis.
+            // You should not init your app in this process.
+            return;
+        }
+        LeakCanary.install(this);
+        // Normal app init code...
```

Comment:
Can you remove this comment? It does not seam relevant to this code because the app is being initialized elsewhere

Reply:
Done